### ISSUE TITLE
about loss

### ISSUE THREAD

**Issue by hpjmlgy** [hardikbansal/Fader-Networks-Tensorflow#1]
where is the discriminator loss defination？i can only find out the reconstruction error loss defination.

**Comment by hardikbansal** [comment]
I have not completed the code yet. Will finish by the end of week. Will update you when it is done.

**Comment by hpjmlgy** [comment]
copy that. In the paper ,they said they use one-hot to encode the prior,then  the discrimintive loss i thought would be softmax （1-p gona work）. Am i right dude？(•̀⌄•́)

**Comment by hardikbansal** [comment]
I have added the loss now. You can check it. I have not yet tested the model, but i am nearly done with it.
